src/sass/components/_tour.scss

```diff
@@ -0,0 +1,31 @@
+// Tour
+// ====
+
+// Overriding inline styles to add changes to hover and focus states
+
+/* stylelint-disable declaration-no-important */
+.react-joyride__tooltip {
```

Comment:
Wouldn't it be cool if we had CSS layers?

Reply:
It would have come in really handy.